tox.ini

```diff
@@ -84,7 +80,7 @@ commands =
 description = Run integration tests
 deps =
     pytest
-    juju
+    juju==2.9.11
```

Comment:
Yes, I also noticed that this was needed for tests to pass, when upgrading to ops=2.0.0rc2. 

We should really go for poetry. It will make builds and tests more stable! I will open a PR across all kafka repos in the coming days